askem.dbml

```diff
@@ -92,6 +92,7 @@ Table dataset {
   geospatial_resolution varchar
   annotations json
   maintainer int [not null, ref: > public.person.id]
+  simulation_run bool
```

Comment:
```suggestion
  simulation_run bool [default: false]
```
could we default it to false? if doesn't work here, it could be defaulted to false in `tds/schema/simulation.py`